modules/features/cu_hero_unit/cu_hero_unit.features.field_base.inc

```diff
@@ -143,6 +143,32 @@ function cu_hero_unit_field_default_field_bases() {
     'type' => 'list_text',
   );
 
+  // Exported field_base: 'field_hero_unit_overlay'.
+  $field_bases['field_hero_unit_overlay'] = array(
+    'active' => 1,
+    'cardinality' => 1,
+    'deleted' => 0,
+    'entity_types' => array(),
+    'field_name' => 'field_hero_unit_overlay',
+    'indexes' => array(
+      'value' => array(
+        0 => 'value',
+      ),
+    ),
+    'locked' => 0,
+    'module' => 'list',
+    'settings' => array(
+      'allowed_values' => array(
```

Comment:
I tried out some different setting for this for using white & black text.

75% (for black text/white overlay) and 50% (for white text/black overlay) seemed to work best with the range of images I tried.  I think if we open it to a range of values that we could get cases where people think that they've made their content accessible, but really haven't.  

If people want more control, I would push them to edit the image beforehand.